main/zoe/api/zoe-contract-facet.md

```diff
@@ -415,39 +424,29 @@ a valid keyword, or is not unique.
 ```js
 zcf.assertUniqueKeyword(keyword);
 ```
-## `zcf.reallocate(seatStagings)`
-- `seatStagings` `{SeatStaging[]}` (at least two)
+## `zcf.reallocate(seats)`
+- `seats` `{ZCFSeats[]}` (at least two)
 - Returns: `{void}`
 
-The contract reallocates over `seatStagings`, which are
-associations of `seats` with new `allocations` to be used in reallocation.
-There must be at least two `seatStagings` in the array argument.
+The contract reallocates over its `seats` arguments, which are
+associations of `seats` with staged allocations to be used in reallocation.
```

Comment:
```suggestion
The contract reallocates over its `seats` arguments.
```

Reply:
Tightened a bit. How's this:

`zcf.reallocate()` commits the staged allocations for each of its seat arguments,
making their staged allocations their current allocations. `zcf.reallocate()` then
transfers the assets escrowed in Zoe from one seat to another. Importantly, the assets 
stay escrowed, with only the internal Zoe accounting of each seat's allocation changed.

Reply:
Sounds great!